stellargraph/layer/watch_your_step.py

```diff
@@ -181,13 +181,29 @@ def __init__(
 
         self.embedding_dimension = embedding_dimension
 
-        self.attention_regularizer = attention_regularizer
-        self.attention_initializer = attention_initializer
-        self.attention_constraint = attention_constraint
-
-        self.embeddings_initializer = embeddings_initializer
-        self.embeddings_regularizer = embeddings_regularizer
-        self.embeddings_constraint = embeddings_constraint
+        self._left_embedding = Embedding(
```

Comment:
Just reviewed you ComplEx and Distmult stateful PR - in that PR the embeddings are not hidden but here there are. Can you say why you went for different approaches?

Reply:
No particular reason. I will update #1100.